lib/task_details_page.dart

```diff
@@ -19,6 +19,7 @@ class TaskDetailsPage extends StatefulWidget {
   _TaskDetailsPageState createState() => _TaskDetailsPageState();
 }
 
+// Carlo
 class _TaskDetailsPageState extends State<TaskDetailsPage> {
   late TextEditingController _titleController;
```

Comment:
Initialising before would let you delete this line of code (line 27): 
  _TaskDetailsPageState() : dueDate = DateTime.now();  // default initialization